diff --git a/apps/api/api/server.ts b/apps/api/api/server.ts
index e3653f2..1746357 100644
--- a/apps/api/api/server.ts
+++ b/apps/api/api/server.ts
@@ -27,7 +27,7 @@ if (envPath) {
 /**
  * start server with port
  */
-const PORT = process.env.PORT || 3001;
+const PORT = Number(process.env.PORT || 3001);
 const HOST = process.env.HOST || (process.env.NODE_ENV === 'production' ? '127.0.0.1' : '0.0.0.0');
 
 const startServer = async () => {
diff --git a/apps/api/package.json b/apps/api/package.json
index 044931f..50ab9f9 100644
--- a/apps/api/package.json
+++ b/apps/api/package.json
@@ -31,7 +31,8 @@
     "nodemailer": "^9.0.3",
     "pm2": "^7.0.1",
     "sequelize": "^6.37.5",
-    "tsx": "^4.20.3"
+    "tsx": "^4.20.3",
+    "zod": "^3.24.1"
   },
   "devDependencies": {
     "@types/bcrypt": "^5.0.2",
